libs/core-ui/src/lib/Interfaces/ExplanationInterfaces.ts

```diff
@@ -33,8 +34,8 @@ export interface ISerializedExplanationData {
 }
 
 export interface IModelInformation {
```

Comment:
it might be better to move this to the other model interface file, since this related to model

Reply:
This is one of several interfaces I want to completely get rid of. See the comment below this interface and at the top of the file. IDataset and IModelExplanationData should have everything we need. It's just a matter of migrating to that set of cleaner interfaces.